mantidproject/mantid
# Move unscripted testing from user-docs to dev-docs

This PR moves the unscripted tests for Muon from the user docs to the dev-docs

**To test:**
1. Build dev-docs.
1. Check built docs in testing.
1. Make sure the contents of `MuonAnalysis_test_guides` have been built correctly
1. Make sure `Engineering Diffraction Test Guide` have been built correctly
1. Build user docs
1. It should build without errors

Fixes #27199

*This does not require release notes* because It is a monor change in docs that users would not be looking into

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
